migrations/v2/V202408201553__v2.4.x__ContentAutotagging.sql

```diff
@@ -318,15 +321,1513 @@ EXEC [${flyway:defaultSchema}].spUpdateEntityFieldRelatedEntityNameFieldMap @Ent
+--INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B693AD50-0E66-EF11-A752-C0A5E8ACCB22', 14, 'ContentType', 'Content Type', NULL, 'nvarchar', 510, 0, 0, 0, 'null', 0, 0, 1, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
+--INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B693AD50-0E66-EF11-A752-C0A5E8ACCB22', 15, 'ContentSourceType', 'Content Source Type', NULL, 'nvarchar', 510, 0, 0, 0, 'null', 0, 0, 1, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
+--INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B693AD50-0E66-EF11-A752-C0A5E8ACCB22', 16, 'ContentFileType', 'Content File Type', NULL, 'nvarchar', 510, 0, 0, 0, 'null', 0, 0, 1, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
+--INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'F13EC656-0E66-EF11-A752-C0A5E8ACCB22', 7, 'ContentItem', 'Content Item', NULL, 'nvarchar', 500, 0, 0, 1, 'null', 0, 0, 1, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
+--INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'F63EC656-0E66-EF11-A752-C0A5E8ACCB22', 6, 'Item', 'Item', NULL, 'nvarchar', 500, 0, 0, 1, 'null', 0, 0, 1, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' )
+
+INSERT INTO [${flyway:defaultSchema}].ContentFileType ( Name, FileExtension ) VALUES ( 'PDF', '.pdf' )
```

Comment:
Does the `ContentFileType` table have an `ID` column as primary key using `uniqueidentifier` like all the other core entities? If so, we should specify the ID in the INSERT statement